README.md

```diff
@@ -58,6 +58,7 @@ A curated list of awesome open source workflow engines
 * [Titanoboa](https://titanoboa.io/) [![Stars](https://img.shields.io/github/stars/mikub/titanoboa.svg)](https://github.com/mikub/titanoboa) - Titanoboa is a platform for creating complex workflows on JVM.
 * [uTask](https://github.com/ovh/utask) [![Stars](https://img.shields.io/github/stars/ovh/utask)](https://github.com/ovh/utask/stargazers) - Automation engine that models and executes business processes declared in yaml.
 * [Workflow Engine](https://workflowengine.io) - A lightweight .NET and Java workflow engine.
+* [Wexflow - Open Source Workflow Engine](https://github.com/aelassas/wexflow) - Wexflow is an open source and cross-platform workflow engine and automation platform that aims to automate recurring tasks. With the help of Wexflow, building automation and workflow processes become easy.
```

Comment:
note this list is by alphabetic order  , so this should be above `Workflow Engine` one

Reply:
corrected :)